src/main/java/com/redhat/devtools/intellij/common/utils/ExecHelper.java

```diff
@@ -406,17 +431,12 @@ public boolean isConnected() {
         };
       }
 
-      @Override
-      public String runningTargetName() {
-        return null;
-      }
     };
-    return runner;
   }
 
   /**
    * Ensure the terminal window tab is created. This is required because some IJ editions (2018.3) do not
-   * initialize this window when you create a TerminalView through {@link #linkProcessToTerminal(Process, Project, String, boolean)}
+   * initialize this window when you create a TerminalView through {@link #linkProcessToTerminal(PtyProcess, Project, String, boolean)}
```

Comment:
@lstocchi looks like this is not required any more since we move to IC-2019.3?

Reply:
Yes i guess it can be removed